### ISSUE TITLE
Segmentation Mask Scores

### ISSUE THREAD

**Issue by tywei08** [facebookresearch/sam3#299]
I want to get a score for each predicted mask in each frame. I found in the output of predictor.handle_stream_request, the field "out_probs" stays constant across frames. Is "out_probs" just the detection score for the first frame? 
```
for response in predictor.handle_stream_request(request=dict(type="propagate_in_video", session_id=session_id)):
    outputs = response["outputs"]
    obj_score_logits = outputs["out_probs"] # stays the same across frames
    ...
```
Any suggestion of how to get the per-mask scores is appreciated.

**Comment by OrangeSodahub** [comment]
@tywei08 Hi, have you figured it out?
